src/components/BorrowerProfile/LendCta.vue

```diff
@@ -778,6 +778,7 @@ export default {
 			return 'tw-transform tw-translate-y-7 md:tw--translate-y-7 lg:tw--translate-y-7';
 		},
 		isLessThan25() {
+			if (this.enableFiveDollarsNotes) return false;
```

Comment:
I'd recommend adding a comment above this line as to why this should return false for $5 notes.